timer.js

```diff
@@ -1,11 +1,16 @@
 const totalTime = 30;
-let timeLeft = totalTime;
+let timeLeft;
 
 function timer() {
     setTimeout(() => {
         if (timeLeft > 0) {
             timer();
         } else {
+            let playerScore = document.querySelector('#player_score').textContent.replace('Score: ', '');
+
+            document.querySelector('#game_over').style.display = 'flex';
```

Comment:
same here, can we have a function for this? theres 2 occurances of making a screen flex